docs/_themes/clusterhq/layout_docs.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
+
+    <link href="{{ pathto('_static/css/bootstrap.min.css', 1) }}" rel="stylesheet">
+    <link href="{{ pathto('_static/css/' + style, 1) }}" rel="stylesheet">
+    <link href="{{ pathto('_static/css/pygments.css', 1) }}" rel="stylesheet">
+    
+    <script type="text/javascript" src="//fast.fonts.net/jsapi/b20e6cd5-c1a4-4878-9d69-da3806cb86b8.js"></script>
```

Comment:
Hmm, scheme-relative URLs are useful when you don't know what protocol someone will be accessing a page with, but the best technique is simply if an asset is available over HTTPS, you include it via HTTPS. Mixed content warnings should occur only when the browser requested a URL via HTTPS and the document served in the response subsequently requests assets over unencrypted HTTP. You shouldn't get a browser warning when accessing the page over HTTP but internally that document is including some HTTPS assets.

So the short version is: it's preferable to hard-code all the assets in the document as HTTPS than to leave them scheme-relative.